### PR TITLE
Add RFC template

### PR DESCRIPTION
**Pull request by oudod**

### CODE REVIEW

**Review by alexauroradev — APPROVED** [review]
LGTM

File: 0000-template.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+corner-cases, and include examples of how the feature is used.
+
+# How We Teach This
+[how-we-teach-this]: #how-we-teach-this
+
+What names and terminology work best for these concepts and why?
+How is this idea best presented—as a continuation of existing Exonum concepts,
+or as a wholly new one?
+
+Would the acceptance of this proposal change how Exonum is taught to new users
```
Comment (slowli) [inline 0000-template.md]: "Exonum is taught" sounds strangely awesome 😳

File: 0000-template.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+at any level?
+How should this feature be introduced and taught to existing Exonum users?
+
+What additions or changes to the Exonum-doc does it entail?
```
Comment (slowli) [inline 0000-template.md]: Please change to "the Exonum documentation" and link to the repository.

File: 0000-template.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+# Detailed design
+[design]: #detailed-design
+
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with the Exonum to understand, and for somebody familiar with the
```
Comment (slowli) [inline 0000-template.md]: "The Exonum" sounds wrong.

**Review by slowli — COMMENTED** [review]
Mostly good (since it's derived from Rust :). Can we use the RFC template from Rust, btw? (See my comment in #1).

File: 0000-template.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+How is this idea best presented—as a continuation of existing Exonum concepts,
+or as a wholly new one?
+
+Would the acceptance of this proposal change how Exonum is presented to new users
+at any level?
+How should this feature be introduced and taught to existing Exonum users?
+
+What additions or changes to the [Exonum documentation] does it entail?
+
+What would an outline of a new section in the documentation look like? What
```
Comment (Maria-Nosyk) [inline 0000-template.md]: the outline

**Pull request merged**